test/components/select_dropdown_test.js

```diff
@@ -9,9 +9,25 @@ describe("SelectDropdown component", () => {
     value: "value",
     onChange() {},
     selectOptions: [],
+    isRequired: false,
   }
   let wrapper
 
+  context("when the stage is 'idea-generation'", () => {
```

Comment:
need to update the test language here to reflect the new prop!

Reply:
cool - done!